### PR TITLE
Update architecture docs - monitoring

### PR DESCRIPTION
**Pull request by jeffreyscarpenter**
Proposed updates to monitoring architecture page.

### CODE REVIEW

File: docs/content/en/docs/architecture/monitoring/_index.md [hunk 1]
```diff
@@ -6,24 +6,39 @@ description: >
   How do you know if your cluster is healthy?
 ---
 
-When running applications in Kubernetes observability is key. With K8ssandra and cass-operator, each Apache Cassandra® pod is configured with the DataStax [Metrics Collector for Apache Cassandra](https://github.com/datastax/metric-collector-for-apache-cassandra). This exposes Cassandra node-level metrics in the Prometheus format covering everything from operations per second and latency to compaction throughput and heap usage.
+When running applications in Kubernetes, observability is key. K8ssandra includes Prometheus and Grafana for the storage and visualization of metrics associated with the Cassandra cluster.
+
+![Monitoring Overview](monitoring-overview.png)
+
+With K8ssandra and cass-operator,  This exposes Cassandra node-level metrics in the Prometheus format, covering everything from operations per second and latency, to compaction throughput and heap usage. Examples of these metrics are shown in the Grafana dashboard below.
```
Comment (johnwfrancis) [inline docs/content/en/docs/architecture/monitoring/_index.md:13]: Looks like a fragment...

**Review by johnwfrancis — CHANGES_REQUESTED** [review]

File: docs/content/en/docs/architecture/monitoring/_index.md (hunk 1 above)
Comment (jeffreyscarpenter) [reply docs/content/en/docs/architecture/monitoring/_index.md]: updated to remove fragment

**Review by johnwfrancis — APPROVED** [review]

**Review by bradfordcp — APPROVED** [review]

File: docs/content/en/docs/architecture/monitoring/_index.md [hunk 2]
```diff
@@ -6,24 +6,39 @@ description: >
...
+![Monitoring Architecture](monitoring-architecture.png)
+
+Let's walk through this architecture from left to right. We'll provide links to the Kubernetes documentation so you can dig into those concepts more if you'd like to.
 
-* Prometheus `ServiceMonitor` resource with Metric Relabeling directed at the
-  K8ssandra cluster's Service
-* Grafana `DataSource` resource configured to reference the deployed Prometheus
-  instance
-* Grafana `Dashboard` resources
+* The Cassandra nodes in a K8ssandra-managed cluster are organized in one or more data centers, each of which is represented as a Kubernetes [StatefulSet](https://kubernetes.io/docs/concepts/workloads/controllers/statefulset/). (We'll focus here on details of the Cassandra node related to monitoring, you can see other details about Cassandra nodes such as how storage is managed on the [Cassandra architecture](/docs/architecture/cassandra) page.)
```
Comment (bradfordcp) [inline docs/content/en/docs/architecture/monitoring/_index.md:25]: Each _rack_ is a stateful set. A 3 rack DC will be comprised of 3 stateful sets. Also, "data center" is considered a single word "datacenter".

Comment (bradfordcp) [reply docs/content/en/docs/architecture/monitoring/_index.md:25]:
```suggestion
* The Cassandra nodes in a K8ssandra-managed cluster are organized in one or more datacenters, each of which is composed of one or more racks. Each rack represents a failure domain with replicas being placed across multiple racks (if present). In Kubernetes, racks are represented as [StatefulSets](https://kubernetes.io/docs/concepts/workloads/controllers/statefulset/). (We'll focus here on details of the Cassandra node related to monitoring, you can see other details about Cassandra nodes such as how storage is managed on the [Cassandra architecture](/docs/architecture/cassandra) page.)
```

File: docs/content/en/docs/architecture/monitoring/_index.md [hunk 3]
```diff
@@ -6,24 +6,39 @@ description: >
...
 
-## Metric Data Retention
+* One or more Prometheus instances are deployed in another StatefulSet, with the default configuration starting with a single instance. Using a StatefulSet allows each Prometheus node to connect to a Persistent Volume (PV) for longer term storage. The default K8ssandra chart configuration does not use PVs. By default, metric data collected in the cluster is retained within Prometheus for 24 hours.
 
-Metric data collected in the cluster is retained within Prometheus for 24 hours.
+* An instance of the Prometheus Operator is deployed using a Replica Set. The `kube-prometheus-stack` also defines several useful Kubernetes [custom resources (CRDs)](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/) that the Prometheus Operator uses to manage Prometheus. One of these is the `ServiceMonitor`. K8ssandra uses `ServiceMonitor` resources, specifying labels selectors to indicate the Cassandra nodes to connect to in each datacenter, and how to relabel each metric as it is stored in Prometheus. `ServiceMonitors` can also be configured to pull metrics from `cass-operator` and Stargate.
```
Comment (bradfordcp) [inline docs/content/en/docs/architecture/monitoring/_index.md:31]: We provide a service monitor for Stargate when it is enabled. I don't know if you want to update this text based on that information.

Comment (bradfordcp) [reply docs/content/en/docs/architecture/monitoring/_index.md:31]:
```suggestion
* An instance of the Prometheus Operator is deployed using a Replica Set. The `kube-prometheus-stack` also defines several useful Kubernetes [custom resources (CRDs)](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/) that the Prometheus Operator uses to manage Prometheus. One of these is the `ServiceMonitor`. K8ssandra uses `ServiceMonitor` resources, specifying labels selectors to indicate the Cassandra pods to connect to in each datacenter, and how to relabel each metric as it is stored in Prometheus. K8ssandra provides a `ServiceMonitor` for Stargate when it is enabled. Users may also configure `ServiceMonitors` to pull metrics from the various operators, but pre-configured instances are not provided at this time.
```

File: docs/content/en/docs/architecture/monitoring/_index.md [hunk 4]
```diff
@@ -6,24 +6,39 @@ description: >
...
+
+* The `AlertManager` is an additional resource provided by `kube-prometheus-stack` that can be configured to specify thresholds for specific metrics that will trigger alerts. Use of this feature is a K8ssandra [roadmap](/docs/roadmap) item.
```
Comment (bradfordcp) [inline docs/content/en/docs/architecture/monitoring/_index.md:33]: Users may enable, and configure, AlertManager through the `values.yaml` file. Maybe we should direct them to check the options out at https://github.com/prometheus-community/helm-charts/blob/main/charts/kube-prometheus-stack/values.yaml#L114-L595?

**Review by bradfordcp — CHANGES_REQUESTED** [review]

**Comment by bradfordcp** [comment]
Should the lower half of this page be bulleted? Is there a reason we're not using paragraphs?

**Review by bradfordcp — APPROVED** [review]

**Pull request merged**